### PR TITLE
Add ability to adjust fuzzy matching thresholds

### PR DESCRIPTION
**Pull request by ehwenk**
changes made to match_taxa.R:
* add fuzzy matching "sliders" to set levels of fuzziness (towards #73)
* turn off fuzzy matches (towards #12)
* turn off imprecise fuzzy matching (set as default)

These changes all work in the function `match_taxa`. However, only the defaults are carried through to `align_taxa`.

### CODE REVIEW

File: tests/testthat/consistency_lookup.csv
Comment (fontikar) [inline tests/testthat/consistency_lookup.csv:1]: Hi @ehwenk, 

it looks like you committed consistency.csv at the root of the directory, could you move to tests/testthat/  - this is triggering a WARNING in R CMD check

File: R/align_taxa.R
Comment (fontikar) [inline R/align_taxa.R:1]: Hi @ehwenk after any changes to the documention e.g roxygen could you please run `devtools::document` so devtools can rebuild the help files - this is triggering a NOTE in R CMD check

File: R/align_taxa.R [hunk 1]
```diff
@@ -119,7 +132,7 @@ align_taxa <- function(original_name,
   )
   
   # do the actual matching
-  taxa <- match_taxa(taxa, resources)
+  taxa <- match_taxa(taxa, resources, fuzzy_abs_dist, fuzzy_rel_dist, fuzzy_matches, imprecise_fuzzy_matches, APNI_matches = FALSE, dataset_id)
```
Comment (fontikar) [inline R/align_taxa.R:135]: I think putting `APNI_matches = FALSE` here will overwrite any upstream APNI_matches = TRUE if specificed in align_taxa() is this an intended behaviour? if not, I think putting `APNI_matches` will do. The default is set in match_taxa already

File: R/align_taxa.R
Comment (ehwenk) [reply R/align_taxa.R:1]: Sorry, didn't see this before. Hopefully now it will pass all tests

**Review by fontikar — APPROVED** [review]
Yay well done!!

**Pull request merged**